domains/emissions/Emissions.API/Tests/Resources/datasync_measurements.json

```diff
@@ -1,16 +1,16 @@
 [
-    {
-        "gsrn": "571313121223234323",
-        "dateFrom": "1609455600",
-        "dateTo": "1609459200",
-        "quantity": 1250,
-        "quality": "measured"
-    },
-    {
-        "gsrn": "571313121223234323",
-        "dateFrom": "1609459200",
-        "dateTo": "1609462800",
-        "quantity": 2520,
-        "quality": "measured"
-    }
-]
+  {
+    "gsrn": "571313121223234323",
+    "dateFrom": "1609455600",
+    "dateTo": "1609459200",
+    "quantity": 1250,
+    "quality": "measured"
+  },
+  {
+    "gsrn": "571313121223234323",
+    "dateFrom": "1609459200",
+    "dateTo": "1609462800",
+    "quantity": 2520,
+    "quality": "measured"
+  }
+]
```

Comment:
You are only changing formatting, we might have to align on this short of thing in the future.

Reply:
This was literally copy/pasted from the old version of the file, I didn't change any formatting on purpose :-P